routes/usersRouter.js

```diff
@@ -0,0 +1,87 @@
+require('dotenv').config()
```

Comment:
Maybe move this to `/index.js` (i.e. where the express server "starts"). Once it's in-scope (higher level), we never need to call this line again.